CEP 10: Conda Version Support (#25)
conda-incubator/ceps: new-cep.md

Reviewer: @travishathaway Hey, can you rename this to `cep-10.md` please?

<table>
<tr><td> Title </td><td> Conda Version Support </td>
<tr><td> Status </td><td> Draft </td></tr>
<tr><td> Author(s) </td><td> Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>
<tr><td> Created </td><td> March 23, 2023</td></tr>
<tr><td> Updated </td><td> March 23, 2023</td></tr>
<tr><td> Discussion </td><td>  NA </td></tr>
<tr><td> Implementation </td><td> NA </td></tr>
</table>

## Abstract

This CEP introduces an official version support policy to promote transparency and certainty 
about which versions of conda are supported. The policy itself states that only the latest

> released version of conda will be supported. Instead of patching previous releases, the core
> development team will only focus its efforts on new and current releases. The implications of

Reviewer 2: ```suggestion
released version of conda will be supported. Instead of patching previous releases, maintainers will only focus their efforts on new and current releases. The implications of
```

users the impression that every version of conda will be supported indefinitely.
Therefore, we felt it necessary to outline with this CEP exactly which versions
will be supported and how. This not only helps set expectations for our users but 
also eases our development efforts knowing we no longer have to support older versions 
of conda.

## Rationale

For many projects (more information here: https://endoflife.date), either
the latest version is supported or the projects have a sliding window of time
for their supported versions. This window is a guarantee saying that the 
version in question will be supported for a specific amount of time. For most who
use a version window, they further specify the types of fixes a version will receive
as it ages.

For conda, we decided to not pursue supporting a sliding window of versions. Instead,
we decided to only support the latest version. One reason is that it is relatively easy
for users to update conda. The second reason is that the development team simply does not
have the resources available to support more than one version at a time. Adding
multiple versions that need to be supported would be burdensome and negatively impact

> momentum on future releases.

Author: 👏  thanks for the improvement.

<tr><td> Title </td><td> Conda Version Support </td>
<tr><td> Status </td><td> Draft </td></tr>
<tr><td> Author(s) </td><td> Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>

> <tr><td> Created </td><td> March 23, 2023</td></tr>

Reviewer 3: Not sure if you meant to change the creation date along with the update date.

```suggestion
<tr><td> Created </td><td> May 19, 2022</td></tr>
```

Author: Thanks!
